internal/runners/auth/auth.go

```diff
@@ -62,7 +62,7 @@ type SignupParams struct {
 func (a *Auth) Run(params *AuthParams) error {
 	if !a.Authenticated() {
 		if err := params.verify(); err != nil {
-			return locale.WrapInputError(err, "err_auth_params", "Invalid authentication params")
```

Comment:
Sorry I missed this on the last review. Why is this no longer an input error?

Reply:
It is already an input error, as returned by params.verify(). It's an extra call that doesn't need to be made.